web/annotation_layer_builder.css

```diff
@@ -52,6 +53,19 @@
   width: 100%;
 }
 
+.annotationLayer .textWidgetAnnotation input[disabled],
+.annotationLayer .textWidgetAnnotation textarea[disabled] {
+  background: none;
+  border: 1px solid transparent;
+  cursor: not-allowed;
+}
+
+.annotationLayer .textWidgetAnnotation textarea {
```

Comment:
Done in the new commit. I thought I had this, but it somehow got lost. Thanks for noticing!